### DISCUSSION TITLE
Cuentamos tus ideas

### DISCUSSION THREAD

**Discussion by camilohuertas02** [equipofisicaud/fisicaud#4] [General]
Que podemos agregar y mejorarle a la pagina web?

**Comment by AnaMaria8903** [comment]
Sugerencias para un buen diseño:

1. Tenemos que tener en cuenta que el fondo no puede ser de un color muy fuerte ya que perturba mucho la vista y recordemos que hay personas con defectos visuales. Tampoco se aconseja un fondo con imagen ya que esto distrae mucho la lectura y altera el color de la tipografía, situación que resulta molesta. 

2. Si quieren un fondo con una imagen, les recomiendo que la imagen tenga una opacidad entre el 10 y el 25%, es decir, marca de agua. 

3. Sugiero que se usen máximo dos tipografías distintas en la página web. Estas deben ser claras, sin muchos adornos ni recargadas para así facilitar la lectura entre los internautas. 

4. Ojo con los derechos de autor. Las tipografías empleadas deben ser de uso libre, ya que si usamos tipografías protegidas por derechos de autor, nos podemos meter en líos judiciales. Lo mismo con el uso de las imágenes. 

5. El uso de imágenes y videos debe ser moderado, ya que si se recarga mucho el sitio web con estos elementos, la página se tornará lenta para cargar, consumirá mucho ancho de banda y esto entorpece la labor del equipo de pruebas.
